src/Application/Fees/Calculator.php

```diff
@@ -26,11 +27,62 @@ class Calculator
     public const STRIPE_CARD_BRANDS = [
         'amex', 'diners', 'discover', 'eftpos_au', 'jcb', 'mastercard', 'unionpay', 'visa', 'unknown'
     ];
+    private const STRIPE_FEES = [
+        // Based on Stripe support email 9/4/21.
+        'fixed' => [
+            'CHF' => '0.3',
+            'DKK' => '1.8',
+            'EUR' => '0.25',
+            'GBP' => '0.2', // Baseline fee in pounds
+            'NOK' => '1.8',
+            'SEK' => '1.8',
+            'USD' => '0.3',
+            'default' => '0.2',
+        ],
+        'gift_aid_percentage' => '0.75', // 3% of Gift Aid amount.
+        'main_percentage_standard' => '1.5',
+        'main_percentage_amex_or_non_uk_eu' => '3.2',
+        // The rate at which VAT is either being or is about to be charged.
+        'vat_percentage_live' => '20',
```

Comment:
Rename to make the comment redundant, then delete the redundant comment.
```suggestion
        'rate_at_which_VAT_is_either_being_or_is_about_to_be_charged' => '20',
```

Reply:
Thanks for taking a look @JayBazuzi . This section of code is moved unchanged from settings.php in this PR, and as there isn't a business requirement right now to change how it works (only to change *when* it runs) I don't want to take time to edit inside this code much now. This PR is more about changing where this code sits in relation to the rest of the matchbot system.

Reply:
If we were going to edit it we'd just remove the "about to" thing, since it's a reference to a change that happened in the past.